SetupDataPkg/Library/ConfigKnobShimLib/ConfigKnobShimDxeLib/GoogleTest/ConfigKnobShimDxeLibGoogleTest.inf

```diff
@@ -0,0 +1,34 @@
+## @file
+# Unit test suite for the ConfigKnobShimDxeLib using Google Test
+#
+# Copyright (c) 2022, Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION         = 0x00010017
+  BASE_NAME           = ConfigKnobShimDxeLibGoogleTest
+  FILE_GUID           = 95B1405E-10E8-4667-84CC-1C2B0A06C424
+  VERSION_STRING      = 1.0
+  MODULE_TYPE         = HOST_APPLICATION
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64
+#
+
+[Sources]
+  ConfigKnobShimDxeLibGoogleTest.cpp
+
+[Packages]
+  MdePkg/MdePkg.dec
+  SetupDataPkg/SetupDataPkg.dec
+  UnitTestFrameworkPkg/UnitTestFrameworkPkg.dec
+
+[LibraryClasses]
+  GoogleTestLib
+  BaseLib
+  DebugLib
+  ConfigKnobShimLib
+  
```

Comment:
This looks like maybe you have some extra whitespace on line 34, which is why it doesn't think there is a newline at the EOF